css/main.css

```diff
@@ -0,0 +1,144 @@
+body {
+	padding: 0;
+	margin: 0;
+	background: #ffffff;
+}
+/*--- navigation bar ---*/
+.navbar {
+	background: #ffffff;
+}
+.navbar-collapse {
+    justify-content: flex-end;
+    background: #ffffff;
+    padding-bottom: 15px;
+    -webkit-transition: all 0.4s cubic-bezier(0.17, 0.25, 0.56, 1.03);
+    transition: all 0.4s cubic-bezier(0.17, 0.25, 0.56, 1.03);
+    border-radius: 5px;
+    margin-top: 5px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDGCohort2/team-040-frontend/pullRequest?prid=6093393)